netdata.spec.in

```diff
@@ -23,6 +32,9 @@ License:	GPL v3+
 Group:		Applications/System
 Source0:	http://firehol.org/download/netdata/releases/v@PACKAGE_VERSION@/%{name}-@PACKAGE_VERSION@.tar.xz
 URL:		http://my-netdata.io/
+%if 0%{?suse_version}
+BuildRequires:	systemd-rpm-macros
```

Comment:
Yes, it is required. I've placed it in the macro conditionals at the top of the page.